packages/integration-tests/src/entities/Author.ts

```diff
@@ -65,6 +65,7 @@ export class Author extends AuthorCodegen {
 }
 
 authorConfig.cascadeDelete("books");
+authorConfig.cascadeDelete("image");
```

Comment:
Ah, great that our made-up-schema already had a "free to be used for this use case" relation to use. :-)